search_engine_parser/core/base.py

```diff
@@ -31,27 +31,24 @@ class SearchItem(dict):
 
 class SearchResult():
     # Hold the results
-    results = []
```

Comment:
The `SearchResult` class was included to provide API compatibility with older versions. Overriding it the way you did would break existing code. In future, after updates, the older ones will enter deprecation. So leave the class alone as it is for now